src/operator/skip.ts

```diff
@@ -8,8 +8,8 @@ import { TeardownLogic } from '../Subscription';
  *
  * <img src="./img/skip.png" width="100%">
  *
- * @param {Number} the `n` of times, items emitted by source Observable should be skipped.
- * @return {Observable} an Observable that skips values emitted by the source Observable.
+ * @param {Number} n - The number of times, items emitted by source Observable should be skipped.
```

Comment:
I know what you are referring to :smiley:

The thingis that the docs are generated based on the ESDoc tags (apparently), not the actual param names in code. So currently, people don't see `total` at all.

Here is what they see currently:

> public skip(the: Number): Observable
>
> ##
> Returns an Observable that skips `n` items emitted by an Observable.
> 
> Params:
>
> Name | Type | Attribute | Description
> -|-|-|-
> the | Number |  | `n` of times, items emitted by source Observable should be skipped.

Here is what they will see with this PR (in its current state):

> public skip(n: Number): Observable
>
> ##
> Returns an Observable that skips `n` items emitted by an Observable.
> 
> Params:
>
> Name | Type | Attribute | Description
> -|-|-|-
> n | Number |  | The number of times, items emitted by source Observable should be skipped.

Here is what they see with your suggestion:

> public skip(total: Number): Observable
>
> ##
> Returns an Observable that skips `n` items emitted by an Observable.
> 
> Params:
>
> Name | Type | Attribute | Description
> -|-|-|-
> total | Number |  | The number of times, items emitted by source Observable should be skipped.

In this last version, the description mentions `n`, which is not refernced anywhere else, so my question was: Should I update the description too, so that it doesn't mention `n` but `total`.

##
And now that I have written this long answer, it seems redundant - it would be simpler if I went ahead and did the change and wait for feedback :grin:
<sub>(So, that's what I'm gonna do...)</sub>